### PR TITLE
OTG - TE-11.1 and TE-11.2 tests update

### LINKED ISSUE #588: OTG: TE-11.1: Backup NHG: Single NH

### LINKED ISSUE #646: OTG: TE-11.2: Backup NHG: Multiple NH

### PR DESCRIPTION
**Pull request by octpetre**
- corrected flapInterface in 11.2
- synced few changes from the ATE test in 11.2
- removed extra waitForARP in 11.1 which causes tests fails when certain interfaces are down
Resolves #646 
Resolves #588

### CODE REVIEW

File: feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go [hunk 1]
```diff
@@ -353,17 +354,19 @@ func (a *testArgs) createFlow(t *testing.T, name, dstMac string) string {
 
 	flow := a.top.Flows().Add().SetName(name)
 	flow.Metrics().SetEnable(true)
-	flow.Size().SetFixed(300)
```
Comment (jasdeep-hundal) [inline feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go:356]: Why the rate change here? ATE tests does use 300 packets.

Also, ATE test is using a range of IPs as the dst address, can we make that chance as well?

File: feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go
Comment (jasdeep-hundal) [inline feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go:1]: Requesting addition of the ATE/DUT port shutdown via use of the port control deviation.

File: feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go
Comment (jasdeep-hundal) [inline feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go:1]: Please re-enabled the "Disable ATE port-2" test case using the ATE port control deviation.

**Review by jasdeep-hundal — CHANGES_REQUESTED** [review]

File: feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go
Comment (octpetre) [reply feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go:1]: Done

File: feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go
Comment (octpetre) [reply feature/experimental/backup_nhg/otg_tests/backup_nhg_test/backup_nhg_test.go:1]: Done

File: feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go (hunk 1 above)
Comment (jasdeep-hundal) [reply feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go:356]: @ANISH-GOTTAPU : Please make these changes and also incorporate the changes from https://github.com/openconfig/featureprofiles/pull/1735

Comment (octpetre) [reply feature/gribi/otg_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go:356]: Done

**Review by jasdeep-hundal — APPROVED** [review]

**Pull request merged**